DigitalEcosystem/exploratory/initial_work/initial_bg_work_commentated/bg_extra_descriptors.py

```diff
@@ -1,18 +1,35 @@
 #!/usr/bin/env python
```

Comment:
We decided to instead keep a different file for the bandgap initial work. This file is commented, but it was more appropriate to move this work into this branch, since it's really just another file we're moving.